src/components/App/index.js

```diff
@@ -184,6 +185,7 @@ export class App extends Component {
 			sound,
 			alerts,
 			modal,
+			customTitle,
```

Comment:
It's not a good idea propagating a new custom property through the containers and components, we can do something simpler, see:
- Since we'll use the `setTheme` method to store the custom title value, which will be stored inside the `iframe.theme` object, you can extract this information in the `Consumer` of the [chat/container.js](https://github.com/RocketChat/Rocket.Chat.Livechat/blob/develop/src/routes/Chat/container.js#L352) file and set it on the `<ChatContainer>` properties:

```
    ...
    title={customTitle || title || I18n.t('Need help?')}
    ...
```
Did you get it?
We won't need to do anything else, just update this prop.

Reply:
This made me fully understand how props are passed between components <3